docs/rules/no-unused-expressions.md

```diff
@@ -46,7 +58,11 @@ The following patterns are not considered problems by default:
 ```js
 /*eslint no-unused-expressions: 2*/
 
-{}
+{} // In this context, this is a block statement, not an object literal
+
+{myLabel: anExpression()} // In this context, this is a block statement with a label and expression, not an object literal
+
+function aFunctionDeclaration () {}
```

Comment:
Yes, this section is showing those that are *not* problems. I wanted to add the function declaration specifically because, as with the `{}` and `{myLabel: anExpression()}` structures (and with directives), the same structure would no longer be a declaration in an expression context.